app/models/v3/persistence/isolation_segment_model.rb

```diff
@@ -1,20 +1,32 @@
 module VCAP::CloudController
   class IsolationSegmentModel < Sequel::Model(:isolation_segments)
+    SHARED_ISOLATION_SEGMENT_GUID = '933b4c58-120b-499a-b85d-4b6fc9e2903b'.freeze
+
     include Serializer
     ISOLATION_SEGMENT_MODEL_REGEX = /\A[[:print:]]+\Z/
 
     one_to_many :spaces,
       key: :isolation_segment_guid,
       primary_key: :guid
 
+    many_to_many :organizations,
+      left_key: :isolation_segment_guid,
+      left_primary_key: :guid,
+      right_key: :organization_guid,
+      right_primary_key: :guid,
+      join_table: :organizations_isolation_segments,
+      after_add: proc { |isolation_segment, org| org.default_isolation_segment(isolation_segment) },
+      before_remove: proc { |isolation_segment, org| org.unset_default_isolation_segment(isolation_segment) }
```

Comment:
We'd like to stick to the same patterns across our v3 code base so our team can have the same expectations for how all associations work.